variables.tf

```diff
@@ -293,3 +293,9 @@ variable "disk_public_network_access" {
   type        = bool
   default     = false
 }
+
+variable "alerts_email_recipients" {
+  description = "List of e-mail recipients for alerts"
+  type = list(string)
+  default = ["tes@ontotext.com"]
```

Comment:
Noooo :laughing:  We don't want the spam!  Leave it an empty `[]`

Reply:
sure